src/oemof/solph/components/_sink.py

```diff
@@ -13,18 +13,57 @@
 SPDX-License-Identifier: MIT
 
 """
+from warnings import warn
 
 from oemof.network import network as on
-
-from oemof.solph._helpers import check_node_object_for_missing_attribute
+from oemof.tools import debugging
 
 
 class Sink(on.Sink):
-    """An object with one input flow and without any constraints."""
+    """A component which is designed for one input flow.
+
+    Parameters
+    ----------
+    label : str
+        String holding the label of the Sink object.
+        The label of each object must be unique.
+
+    Examples
+    --------
+    Defining a Sink:
+
+    >>> from oemof import solph
+    >>> bel = solph.buses.Bus(label='electricity')
+
+    >>> electricity_export = solph.components.Sink(
+    ...    label='el_export',
+    ...    inputs={bel: solph.flows.Flow()})
+
+
+    Notes
+    -----
+    It is theoretically possible to use the Sink object with multiple outputs.
```

Comment:
It should be multiple inputs here, not outputs.

Reply:
Thanks, fixed.